src/hooks/useHandleAttest.tsx

```diff
@@ -550,7 +559,7 @@ async function sui(
           showEvents: true,
         },
       })
-    ).data;
+    ).data as SuiTransactionBlockResponse;
```

Comment:
`as SuiTransactionBlockResponse` is OK here on the `algo()` function ?

Reply:
But this change it's on `sui()` function